pcdsdevices/epics_motor.py

```diff
@@ -1523,6 +1523,35 @@ def __init__(self, prefix='', *, tip_pv, tilt_pv, **kwargs):
         super().__init__(prefix, **kwargs)
 
 
+class SmarActPicoscale(SmarAct):
+    """
+    Class for encoded SmarAct motors controller via the MCS2 controller
+    which use the Picoscale sensor heads for encoded positions.
+    Child of SmarAct device.
+    """
+    # configuration settings and readbacks
+    pico_present = Cpt(EpicsSignalRO, ':PS_PRESENT', kind='config')
```

Comment:
* Could you add some `doc` keyword text describing these?  For example, I'm not immediately clear what the difference between present, exists, and valid is in a practical sense.
* Is the `pico_` variable prefix necessary here?  Or is it obvious enough from the class name?  It seems like the `SmarAct` parent class doesn't really share any of these names except for:
  * `enable`, which collides with the `.enable()`  (maybe `enabled`)?
  * `name`, which collides with the existing `name`  (why not `locator`?)

Reply:
PicoScale is weird and in order for the PicoScale controller to play nice with the MCS2, it needs a combination of states to satisfy the "MCS2 Sensor Enabled" flag that triggers `Channel <N> 0:1 ? (does, does not) have an encoder` which lets you do closed loop movement.

The channel has to be:

1. System stable - ready to produce accurate results for position, (pico_stable)
2. Channel must be enabled for picoscale data (pico_enable)
3. The data must be valid (pico_valid)
4. Beam must not be interrupted

I will gladly add the doc text in the `FCpt` and clean this up :]

I used the pico prefix 80% for ordering the indicators in the resulting detailed/engineering views, 20% for clarifying that it explicitly comes from the PicoScale.

`Locator` is the name of the PV itself so I wanted to keep apples to apples on indicator and associated PV. It's technically the PicoScale IP address, which is given by a node-name. Helpful because we have 3 of these in the FTL and each PicoScale only supports 3 channels on an MCS2. I could change this to `picoscale_name` if it is clearer

Reply:
@aberges-SLAC I think that leaving this as `picoscale_locator` would be preferred by me. That keeps us consistent with the SmarAct naming convention, and these picoscales are unusual enough that if you're using one you probably know what it means. I think that adding a short explanation in the DOC field (which shows up as a tool tip) would go a long way to helping clear up any confusion among those who are not that familiar with this device.

Reply:
Did @aberges-SLAC decide to leave this one as is?

Reply:
Yes, I kept `pico_locator` as it but gave it a better tooltip